Exercise_B/index.css

```diff
@@ -2,3 +2,31 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+
+.box1 {
+    background-color: blue;
+    float: left;
+    width: 100px;
+    height: 100px;
+    display: inline-block;
+
+}
+
+.box2 {
+    background-color: red;
+    float:left;
+    position: relative;
+    margin: 12%;
+    width: 100px;
+    height: 100px;
+    display: inline-block;
+}
+
+.box3 {
+    background-color: green;
+    float: left;
+    width: 100px;
+    height: 100px;
+    display: inline-block;
```

Comment:
I would extract the width and height into a separate class such that you do not repeat the properties over and over again for each block.

Use an ID for the colour and float.
Make a new class that is applied to every html element for the height and width.

Also you are using float wrong, float left with blue, float right with green.
On you red class only have the background colour and:
clear: both;
margin: auto;

Take a look at float again to see how its used.

Though I might add I liked you use of margin 12% to push the element down and the other away from that element.